sui-execution/latest/sui-move-natives/src/object_runtime/mod.rs

```diff
@@ -531,8 +575,10 @@ impl ObjectRuntimeState {
                 (id, (owner, type_, value))
             })
             .collect();
-        for deleted_id in deleted_ids.keys() {
```

Comment:
Woooo I don't think we can delete this yet. This is required at least prior to @tnowacki's fix on the recent child object issue.

Reply:
Good catch! Must have happened in a bad rebase...I'll add this back in!

Reply:
There were no tests that caught this? would it be possible to add one?